languages/JAVA.md

```diff
@@ -33,16 +33,46 @@ The project's site: http://dbeaver.io
 
 The project's site: https://appulse.io
 
+## J
+
+[**JPA**](https://github.com/spring-projects/spring-data-jpa) - is a Java library. **J**ava **P**ersistence **A**PI ensures the management of persistence and object/relational mapping. These are helpful while using the JPA in the development of applications using the platform for Java EE 5. It provides O-R mapping facility to manage relational data in java application.
+
+### JPA areas:
+
+- Java Persistence API
+- O-R mapping metadata
+- The query language
+
+![JPA_LOGO](https://www.ambient-it.net/wp-content/uploads/2016/04/jpa-logo-175.png)
+
+[**Jolly day**](https://github.com/svendiedrichsen/jollyday) - is a Java library. This project solve the holiday 'problem'. The idea behind this is to have an independent API which can be used in any kind of environment. That means no network or database needed.
+
+There are two release branches which are based on different Java versions.
+
+- 0.4.x - Java 7 - uses JODA time API
+- 0.5.x - Java 8
+
+[**JUnit 5**](https://github.com/junit-team/junit5/) - is a Java library. JUnit 5 is the next generation of JUnit. The goal is to create an up-to-date foundation for developer-side testing on the JVM. This includes focusing on Java 8 and above, as well as enabling many different styles of testing.
+JUnit 5 is the result of JUnit Lambda and its crowdfunding campaign on Indiegogo.
+
+![JUnit5_LOGO](https://cloudblogs.microsoft.com/uploads/prod/sites/37/2019/03/JUnit5-hero-image.png)
+
 ## L
 
 [**libGDX**](https://github.com/libgdx/libgdx) is a cross-platform Java game development framework based on OpenGL (ES) that works on Windows, Linux, Mac OS X, Android, your WebGL enabled browser and iOS.
+<br><br><br>
+![libgdx](https://camo.githubusercontent.com/f70f169cab179f13f70e892b3251e6f2032be66d/687474703a2f2f6c69626764782e6261646c6f67696367616d65732e636f6d2f696d672f6c6f676f2e706e67)<br><br>
 
-![libgdx](https://camo.githubusercontent.com/f70f169cab179f13f70e892b3251e6f2032be66d/687474703a2f2f6c69626764782e6261646c6f67696367616d65732e636f6d2f696d672f6c6f676f2e706e67)
+[**Lombock**](https://github.com/rzwitserloot/lombok) Project **Lombok** is a java library that automatically plugs into your editor and build tools, spicing up your java.
```

Comment:
I accidentally ran into this. There is a type here: Should be "Lombok" instead of "Lombock"

Reply:
Yep, small typo 😿